examples/seat-adjuster/requirements-links.txt

```diff
@@ -1 +0,0 @@
-git+https://github.com/eclipse-velocitas/vehicle-app-python-sdk.git@v0.12.2
```

Comment:
Shouldn't we remove the whole file then?

Reply:
Already had this discussion with Markus, some worklows reference the requirements-links.txt - we do not want more changes across our repos. Should be done later

Reply:
Ok - fine